examples/basic_qa_pipeline.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from pathlib import Path
+
+logging.basicConfig(format="%(levelname)s - %(name)s -  %(message)s", level=logging.WARNING)
+logging.getLogger("haystack").setLevel(logging.INFO)
+
+from haystack.document_stores import ElasticsearchDocumentStore
+from haystack.utils import clean_wiki_text, convert_files_to_docs, fetch_archive_from_http, print_answers, launch_es
+from haystack.nodes import FARMReader, BM25Retriever
+from haystack.nodes.file_classifier import FileTypeClassifier
+from haystack.nodes.preprocessor import PreProcessor
+from haystack.nodes.file_converter import TextConverter
+from haystack.pipelines import Pipeline
+from pprint import pprint
+
+
+def basic_qa_pipeline():
+    # launch and create DocumentStore
+    launch_es()
+
+    # Initialize a DocumentStore
+    document_store = ElasticsearchDocumentStore(host="localhost", username="", password="", index="document")
+
+    # fetch, pre-process and write documents
+    doc_dir = "data/basic_qa_pipeline"
+    s3_url = "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-qa/datasets/documents/wiki_gameofthrones_txt1.zip"
```

Comment:
We should rename this dataset and the second one in the other file to enable usage statistics of the scripts with telemetry. For now, I uploaded the two datasets to s3://core-engineering/data/small_faq_covid.csv.zip and s3://core-engineering/data/wiki_gameofthrones_txt1.zip but they don't have the right permission settings yet. @masci I think I would need to edit the bucket policy. Is that okay or could you please have a look? 

@TuanaCelik We also need to decide on a naming scheme. I would be okay with a number as a suffix just like we did until now but maybe we can come up with an alternative? All that's left to do is then a change here: https://github.com/deepset-ai/haystack/blob/ddeaf2c98c157af1e26c637bcb563c6ea52fdcb7/haystack/utils/import_utils.py#L79 and here: https://github.com/deepset-ai/haystack/blob/ddeaf2c98c157af1e26c637bcb563c6ea52fdcb7/haystack/telemetry.py#L208

Reply:
@julian-risch thanks for this. For now, I will add these two datasets you mention here to the PR, ready to be amended if needed. 
Naming scheme-wise, I don't have solid preferences here. But ultimately, if the same dataset is being used in a script as is with a tutorial, I'd like to be able to identify which one it was used in. 
E.g.:
core-engineering/data/tutorial/small_faq_covid.csv.zip
core-engineering/data/scripts/small_faq_covid.csv.zip

Depends on how you want to structure the bucket..

Reply:
The scripts/ and tutorials/ paths are a good idea. 👍

Reply:
I've updated the current urls to the one you messaged over on slack. But I'll leave this conversation live just in case there's a following change :)